community/doc_guidelines/rust_apis.md

```diff
@@ -0,0 +1,40 @@
+# Documentation Guidelines for Rust APIs
+
+<!--
+  Copyright 2018-2020 Cargill Incorporated
+  Licensed under Creative Commons Attribution 4.0 International License
+  https://creativecommons.org/licenses/by/4.0/
+-->
+
+* Follow Splinter's [general documentation guidelines](general.md) and
+  [capitalization guidelines](capitalization.md).
+
+* Provide a crate-level overview (with `//!` comments) that summarizes the
+  purpose of the crate.
+
+* Include a module-level summary (with `//!` comments), using this guideline
+  from [rustlang RFC
+  1574](https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#using-markdown-1):
```

Comment:
Should this be pointing to the markdown link? or should it be pointing to https://github.com/rust-lang/rfcs/blob/master/text/1574-more-api-documentation-conventions.md#module-level-vs-type-level-docs

Reply:
Oops - thanks for catching this.  Fixed.